Makefile

```diff
@@ -1,7 +1,22 @@
-all r4: .setup-cqf-ruler-r4 synthea generate-patients-r4 calculate-patients-r4
-
+MEASURE_DIRS := $(shell ls -d EXM_*)
 export SYNTHEA_DIR := synthea_output/$(shell date +%Y-%m-%dT%H%M%S)
 
+define gen_calc_pts
+	make -C $1 all;
+endef
+
+define load_all_pts
+	./load-cqf-ruler.sh $1 FHIR_VERSION=r4;
```

Comment:
I think we'd want this to call the `make preload-*` targets, since they do other things that are necessary for preloading in addition to calling `load-cqf-ruler.sh`.

Reply:
I'm not sure how well calling other targets is going to work, because of how makefile macros work, but I can essentially do all the steps that the `preload-*` targets do in this target.